src/librustc_session/options.rs

```diff
@@ -935,6 +935,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "run LLVM in non-parallel mode (while keeping codegen-units and ThinLTO)"),
     no_profiler_runtime: bool = (false, parse_no_flag, [TRACKED],
         "prevent automatic injection of the profiler_builtins crate"),
+    disable_unique_symbols: bool = (false, parse_bool, [UNTRACKED],
```

Comment:
@da-x 
Have you seen the comments from the start and from the end of this list?
```
    // This list is in alphabetical order.
    //
    // If you add a new option, please update:
    // - src/librustc_interface/tests.rs
```
(The list of tests is also in alphabetical order.)

Reply:
Fixed in a new commit.